### ISSUE TITLE
Unable to save settings

### ISSUE THREAD

**Issue by mfiano** [Nicotine-Plus/nicotine-plus#32]
Using the latest git release. Upon pressing the Apply button, output to terminal shows:

```
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/pynicotine/gtkgui/frame.py", line 2173, in OnSettingsClosed
    output = self.settingswindow.GetSettings()
  File "/usr/lib/python2.7/site-packages/pynicotine/gtkgui/settingswindow.py", line 3670, in GetSettings
    sub = page.GetSettings()
  File "/usr/lib/python2.7/site-packages/pynicotine/gtkgui/settingswindow.py", line 2101, in GetSettings
    iter_Main = self.PosList.get_iter(self.MainPosition.get_active())
ValueError: invalid tree path
```

**Comment by gfarmerfr** [comment]
Hi @mfiano 

For information, where you upgrading from 1.2.16 to git master or where you already on git before the upgrade ?

Anyway if you close nicotine+ and edit at your ~/.nicotine/config file you should find some tabs position parameters:

tabbrowse = Top
tabinfo = Top
tabmain = Top
tabprivate = Top
tabrooms = Top
tabsearch = Top

Make sure that the first letter of the corresponding value is in uppercase (Top instead of top, Bottom instead of bottom, ...). This should fix the issue.

If you were on master between this commit baa236f (23 Oct) and this commit 8db0f18 (Nov 10) and then update to a post 10 Nov version the bug is triggered. That's a side effect of having to rollback to a value backward compatible with 1.2.16.
I had to make a choice : either a few users using git master will have this issue or everyone upgrading from 1.2.16 to a future release will have it. So I did take the first option.
Sorry for the inconvenience.

**Comment by mfiano** [comment]
@gfarmerfr Neither. I was installing for the first time from git master. I fixed the issue by rolling back to the latest stable release (1.2.16).

**Comment by gfarmerfr** [comment]
@mfiano : i've just fix the issue on git master.
You should now be able to either upgrade or do a clean install from git master without the issue.

Could you confirm that it's working ?
Thx.

**Comment by gfarmerfr** [comment]
@mfiano : this issue should be solved on git master since this commeit 082d9efb1f4fb79440ff4c7d4272ceaa51242d72.

Did you found time to check it out ?

**Comment by mfiano** [comment]
Yes I did and it is fixed. However, I have found many unrelated bugs that I have not had time to report yet. To name a few:

1) Sometimes the Clear Finished/Aborted button does not work. When it eventually happens, it stays broken and you must restart Nicotine.

2) When you have a lot (about 3000 or more) files added to download, the program hangs for several seconds when doing anything, if the Downloads tab is visible. I suspect that it is trying to sort every item each tick of the GUI main loop.

3) Sometimes uploads stop working, with no reason given. It will just sit at "Waiting for peer to connect". A restart of Nicotine fixes this.

4) It takes a few seconds for the Settings window to display when opening it. The program is unresponsive until it does.
